README.md

```diff
@@ -1480,3 +1488,10 @@ client.events.on("retry-after", function(releaseTime) {
 
 > #### Note:
 > Eventually, we would like to automate the retry behaviour for requests. See https://github.com/Kinto/kinto-http.js/issues/34
+
+
+## Upgrading
+
+### From 1.x to 2.x
+
+* `collection.listRecords()` now strips the quotes (`"`) from the `last_modified` attribute in result object (#110).
```

Comment:
I wonder if these instructions shouldn't be part of the release documentation (https://github.com/Kinto/kinto-http.js/releases/tag/v2.0.0), so we could simply link to it here. But it may be good to have the instructions versioned in the repo too. Your call.